pages/reference-smart-account/modules/enableModule.mdx

```diff
@@ -0,0 +1,53 @@
+import { Tabs, Callout } from 'nextra/components'
+
+# `enableModule`
+
+Enables the module `module` for the Safe.
```

Comment:
I'm personally not a big fan of definitions that include the names of the parameters. This applies to many definitions.

"Enables a new Safe Module in a Safe account."